modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -127,6 +132,106 @@ def tx_kill(self, **kwargs):
         res = self.__parse_tx_list(output)
         return res if res else output
 
+    def validate_indexes(self, check_assert: bool = None):
+        """
+        Validate indexes.
+        If check is true, will be return
+        """
+        data = self.__run("--cache validate_indexes")
+
+        if check_assert is not None:
+            assert (('no issues found.' in data) == check_assert), data
+
+        return data
+
+    def idle_verify(self, check_assert: bool = None):
+        """
+        Idle verify.
+        """
+        data = self.__run("--cache idle_verify")
+
+        if check_assert is not None:
+            assert (('idle_verify check has finished, no conflicts have been found.' in data) == check_assert), data
+
+        return data
+
+    def idle_verify_dump(self, node=None, return_path: bool = False):
+        """
+        Idle verify dump.
+        """
+        data = self.__run("--cache idle_verify --dump", node=node)
+
+        if return_path & ('VisorIdleVerifyDumpTask successfully' in data):
+            match = re.search(r'/.*.txt', data)
+            return match[0]
+
+        return data
+
+    def snapshot_create(self, snapshot_name: str, sync_mode: bool = True, time_out: int = 60):
+        """
+        Create snapshot.
+        """
+        res = self.__run(f"--snapshot create {snapshot_name}")
+
+        self.logger.info(res)
+
+        if ("Command [SNAPSHOT] finished with code: 0" in res) & sync_mode:
+            self.await_snapshot(snapshot_name=snapshot_name, time_out=time_out)
+
+        return res
```

Comment:
any reason to have this return?

Reply:
I think the control_utility should return a result

Reply:
fixed